### PR TITLE
Reviewing Opera versions and results, retiring ancient versions

### PR DESCRIPTION
**Pull request by afmenez**
I am adding separate entries for all Opera versions between 7.0 and 12.1.
Due to issue #1127 , I was only able to test with versions 10.50 and later, so no results for previous versions.
Also notice that due to issue #1118 , there are a lot of wrong results on the generated html (there were before, and there are more now).  Actually, I am not sure this patch can be merged until the issue is fixed.

### CODE REVIEW

File: data-es5.js [hunk 1]
```diff
@@ -67,8 +67,7 @@ exports.tests = [
       webkit: true,
       chrome5: null,
       chrome7: true,
-      opera10_50: null,
-      opera12_10: true,
```
Comment (chicoxyzzy) [inline data-es5.js]: Let's keep things explicit where possible (keep `opera12_10: true`) here and elsewhere

**Review by chicoxyzzy — CHANGES_REQUESTED** [review]

File: data-es5.js (hunk 1 above)
Comment (afmenez) [reply data-es5.js]: Sorry, do you want me to add "true" to all tested versions?  This would be a lot of work with little to no gain.

Comment (chicoxyzzy) [reply data-es5.js]: No, just those which was already marked as `true`

**Comment by afmenez** [comment]
Reverted the result consolidation.

**Review by chicoxyzzy — APPROVED** [review]

**Comment by afmenez** [comment]
Notice that even after restoring previous explicit results, this patch still introduces more errors on the generated HTML due to #1118 (there are some wrong values currently).

**Comment by afmenez** [comment]
As I was rebasing this patch, I noticed that it causes some wrong results but it fixes a lot more.  Even with #1118 still pending, I believe this patch is a gain.

**Comment by afmenez** [comment]
Tks!

**Pull request merged**